app/models/rspec-test/spec/user_spec.rb

```diff
@@ -0,0 +1,18 @@
+require_relative '../../user'
+
+RSpec.describe User do
+  describe '#id' do
+    context 'idがnilでない場合' do
+      it 'present?がtrue' do
+        john = User.new(1, 'john@example.com', 'p@ssw0rd')
+        expect(john.present?).to eq true
```

Comment:
Idiomatic

アプリケーションのコードとして実装するときにここまで具体的な人の名前にすることはあんまりないので、
変数名はuserでいいと思った！
（今回はユーザーのインスタンスであることがわかればいい。）

```suggestion
        user = User.new(1, 'john@example.com', 'p@ssw0rd')
        expect(user.present?).to eq true
```